tutorials/robot-marbles-part-1/robot-marbles-part-1.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p>It seems <code>config_sim</code>  can't handle an empty value for the <code>M</code>  key, but it works if <code>M</code> is missing from the <code>dict</code>  . I suggest something like this</p><pre class="ql-syntax" spellcheck="false"># We'll cover the <code>M</code> key in a future article. For now, let's comment it out
simulation_parameters = {
&nbsp;&nbsp;'T': range(10),
&nbsp;&nbsp;'N': 1,
&nbsp;&nbsp;#'M': {}
}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cadCAD-org/cadCAD/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>